tpp-app/src/home/components/TabBarMiddleButton.js

```diff
@@ -0,0 +1,130 @@
+import React, { useRef, useEffect, useState } from 'react';
+import { StyleSheet, TouchableOpacity, Animated, Easing } from 'react-native';
+import { useIsFocused, useNavigation } from '@react-navigation/native';
+import VectorImage from 'react-native-vector-image';
```

Comment:
There is a merge conflict with the develop branch because it's now using the new SVG package rather than this one, so if you change this and just accept the changes from your branch then you should be all good!